content/docs/csidriver/release/unity.md

```diff
@@ -7,7 +7,9 @@ description: Release notes for Unity XT CSI driver
 
 
 
-
+{{% pageinfo color="primary" %}}
+The OpenShift Virtualization platform support is not available for Unity.
```

Comment:
This needs to be removed, we need not call out qualifications exceptions.

Reply:
Addressed.